src/dialogs/quick-bar/ha-quick-bar.ts

```diff
@@ -179,6 +180,12 @@ export class QuickBar extends LitElement {
     this._activatedIndex = 0;
   }
 
+  private _handleSelected(ev: SingleSelectedEvent) {
+    const index = ev.detail.index;
+    const item = (ev.target as any).items[index].item;
+    this.processItemAndCloseDialog(item, index);
+  }
```

Comment:
Is the purpose of this function just to separate the concept of "handling a selection" and "how to execute quick bar commands"?